buildenv/jenkins/benchmarkMetric.py

```
@@ -0,0 +1,27 @@
+import argparse, pathlib, json, re
+
+def main():
+        p = argparse.ArgumentParser()
+        p.add_argument("--console", required=True)
+        p.add_argument("--benchmarkMetricsTemplate_json", required=True)
+        p.add_argument("--fname", required=True)
+        p.add_argument("--testNames", required=True)
+        args = p.parse_args()
+
+        console = pathlib.Path(args.console).read_text(encoding="utf-8")
+
+        benchmarkMetricsTemplate_json = pathlib.Path(args.benchmarkMetricsTemplate_json).read_text(encoding="utf-8")
```

Do we need more error handling around the Filo I/O or just let the script fail?  Will it leave artifacts behind if it does fail?

---

@karianna I believe the script would terminate before the corresponding archive step occurs and the console would report the python error. Errors from read text mean the file can't be accessed, which I'm not sure how to bounce back from with error handling? Later iterations will probably have same issue since I read from same templates. Feels more like a fix code and rerun the script situation to me. @llxia thoughts?